pypeit/coadd2d.py

```diff
@@ -1490,7 +1496,8 @@ def compute_weights(self):
         # adjustment for multislit to case 3) Bright object exists and parset `weights` is equal to 'auto'
         if (self.obj_id_bri is not None) and (self.par['coadd2d']['weights'] == 'auto'):
             # compute weights using bright object
-            _, self.use_weights = self.optimal_weights(self.obj_id_bri, weight_method='constant')
+            # TODO add a parset for weight_method in optimal_weights. The default is currently 'auto'
+            _, self.use_weights = self.optimal_weights(self.obj_id_bri)
```

Comment:
Have you tested how this change from `weight_method = 'constant'` to `'auto'` affects the results?  Or are there any vet tests in place that will do this?